sdks/python/apache_beam/io/avroio_test.py

```diff
@@ -157,7 +158,9 @@ def test_source_display_data(self):
     # No extra avro parameters for AvroSource.
     expected_items = [
         DisplayDataItemMatcher('compression', 'auto'),
-        DisplayDataItemMatcher('file_pattern', file_name)]
+        DisplayDataItemMatcher(
+            'file_pattern',
+            'StaticValueProvider(type=str, value=\'%s\')' % file_name)]
```

Comment:
Please make sure that this DisplayData format matches (in spirit at least) with Java SDK. (sync with @pabloem)